lms/envs/production.py

```diff
@@ -1068,3 +1068,6 @@ def get_env_setting(setting):
 
 ############## DRF overrides ##############
 REST_FRAMEWORK.update(ENV_TOKENS.get('REST_FRAMEWORK', {}))
+
+############################# CELERY ############################
+CELERY_IMPORTS.extend(ENV_TOKENS.get('CELERY_EXTRA_IMPORTS', []))
```

Comment:
Looks great. Could you add documentation for the new setting CELERY_EXTRA_IMPORTS using the how-to I referenced?

Reply:
This is not the doc, but an example: https://github.com/openedx/edx-platform/blob/01bc7277faef73ababd67eeab631e280534b6724/lms/envs/common.py#L71-L74

Reply:
Also, can you confirm you tested this and it is doing what you wish it to do?

Reply:
Sorry. Actually, I wonder if setting the default to [] and the docs for the setting belong in common. That’s where the XBLOCK_EXTRA_MIXIN is first defined.

Reply:
> Also, can you confirm you tested this and it is doing what you wish it to do?

Yes, this change generates the behavior I want; takes the environment variables and adds them to what is in edx-platform

I answer the rest of the comments below